### PR TITLE
Add documentation homepage in French

### PR DESCRIPTION
**Pull request by sieben**

### CODE REVIEW

File: content/fr/docs/home/_index.md [hunk 1]
```diff
@@ -16,4 +16,43 @@ menu:
...
+  button: Voir la référence
+  button_path: /docs/reference
+- name: contribute
+  title: Contribuer à la documentation
+  description: Tout le monde peut contribuer, que vous soyez nouveau dans le projet ou experimenté.
+  button: Contribuer à la documentation
+  button_path: /docs/contribute
+- name: download
+  title: Télécharger Kubernetes
+  description: Si vous installez Kubernetes ou effectuez une mise à niveau vers la version la plus récente, reportez-vous aux notes de version actuelles.
```
Comment (oussemos) [inline content/fr/docs/home/_index.md]: Il manque le `button_path` pour la page de téléchargement.

File: content/fr/docs/home/_index.md [hunk 2]
```diff
@@ -16,4 +16,43 @@ menu:
...
+- name: about
+  title: A propos de la documentation
+  description: Ce site Web contient de la documentation sur les 4 versions actuelles et précédentes de Kubernetes.
```
Comment (oussemos) [inline content/fr/docs/home/_index.md]: Pareil ici

File: content/fr/docs/home/_index.md (hunk 1 above)
Comment (oussemos) [reply content/fr/docs/home/_index.md]: ha, il n'y a pas de button_path sur la page EN

**Comment by sieben** [comment]
Yup :D

Le lun. 25 mars 2019 à 14:56, Oussema CHERNI <notifications@github.com> a
écrit :

**Comment by oussemos** [comment]
Bon, il semble que les liens qui manquent sont gérés d'une autre façon.

/approve

**Comment by zacharysarah** [comment]
@sieben @oussemos Today, 25 March is 1.14 release day. Please don't merge PRs to `master` on release days. We freeze the repo to minimize potential conflicts between `master` and the development branch.

In this case, the freeze was not announced clearly, for which I apologize. This notice is solely to raise visibility about release day freezes.

**Pull request merged**